app/src/main/res/values/strings.xml

```diff
@@ -50,4 +50,7 @@
     <string name="image_download_downloading">Downloading…</string>
     <string name="image_download_success">Image downloaded successfully</string>
     <string name="share_picture">Share picture...</string>
+
+    <!-- TODO: Remove or change this placeholder text -->
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```

Comment:
why ?

Reply:
lol, ça doit rester d'un template d'Android Studio j'imagine

Reply:
remove and I approve

Reply:
et au passage, run lint et essaye de corriger qques trucs, si ont fait tous ça un peu à chaque merge, ça improve la qualité générale du code